types/lib/model.d.ts

```diff
@@ -489,6 +489,8 @@ export interface IndexHintable {
   indexHints?: IndexHint[];
 }
 
+type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>
```

Comment:
This is in conflict with the new `Omit` definition that will ship with the next version of TS, so this line might be removed in the future.